org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -248,6 +250,18 @@ Serializer:
     'serializer' type=STRING
 ;
 
+/////////// Attributes
+Attribute:
+    '@' attrName=AttrName ('(' attrParms+=AttrParm (',' attrParms+=AttrParm)* ')')?
+;
+
+AttrParm:
+    (name=ID '=')? value=AttrParmValue;
```

Comment:
Attributes must be named if there are multiple. Check in the validator.

Reply:
done